### PR TITLE
added react-helmet to handle titles

### PR DESCRIPTION
**Pull request by meinstein**
added helmet to package.json
added a template helmet to root
added individual helmets to all top-level views

### CODE REVIEW

File: src/views/Distributors/index.js [hunk 1]
```diff
@@ -8,30 +9,33 @@ import DistributorMap from './DistributorMap'
...
 
 const Distributors = () => (
-    <QueryRenderer
-        query={graphql`
-            query DistributorsQuery {
-                viewer {
-                    allDistributors {
-                        ...DistributorMap_distributors
-                        ...DistributorList_distributors
+    <>
```
Comment (AlecAivazis) [inline src/views/Distributors/index.js]: I don't think this syntax works with the relay-compiler. Can you check by running `npm run relay` and seeing if it passes without any issues?

Comment (AlecAivazis) [reply src/views/Distributors/index.js]: This should only affect files that have a graphql tag in them i'm pretty sure

File: src/views/Root/index.js [hunk 2]
```diff
@@ -12,6 +13,7 @@ import Nav from './Nav'
 
 export default () => (
     <FlexColumn style={styles.container}>
+        <Helmet titleTemplate="%s | Lark Seeds International" />
```
Comment (AlecAivazis) [inline src/views/Root/index.js]: Can you also add this to the html template? Either in the [index.html](https://github.com/LarkSeeds/catalog/blob/master/src/index.html) file or the [webpack config](https://github.com/LarkSeeds/catalog/blob/master/webpack.config.js#L35) (by setting the `title` argument).

File: src/views/Distributors/index.js (hunk 1 above)
Comment (meinstein) [reply src/views/Distributors/index.js]: yeah, you're right. why though?

Comment (AlecAivazis) [reply src/views/Distributors/index.js]: The relay compiler doesn't use babel v7 so it doesn't have support. I was surprised myself to see it working in the non-relay files but the compiler must look for top level invocations of `graphql` before parsing the rest of the source

File: src/views/Root/index.js (hunk 2 above)
Comment (meinstein) [reply src/views/Root/index.js]: By adding the tag explicitly in the index.html file, are we basically setting a global default?

Comment (AlecAivazis) [reply src/views/Root/index.js]: Yea helmet should take over and replace whatevers in there - this way we have a title while the client file is loading

**Pull request merged**